tools/wallet/src/core/mnemonic.rs

```diff
@@ -6,10 +6,9 @@
 //!
 //! https://github.com/rust-bitcoin/rust-wallet/blob/master/wallet/src/mnemonic.rs
 
-// use crate::error::WalletError;
 use anyhow::{bail, Result};
-// #[cfg(test)]
-// use diem_temppath::TempPath;
+#[cfg(test)]
```

Comment:
let's cleanup the ordering of these imports please. Also for all files in this module. There are cargo tools for reordering imports.

Reply:
Reorganized all imports in the entire module.